<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPrintDaylyReport</class>
 <widget class="QFrame" name="CPrintDaylyReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>报表打印</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(D:/WinParkUI/debug/Image/NewIcon/CommonMiddleBG-normal.jpg);</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>5</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWebView" name="wvReport">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>631</width>
     <height>461</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbType">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>560</y>
     <width>621</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="title">
    <string>报表分类</string>
   </property>
   <widget class="QRadioButton" name="rdChx01">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>19</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>金额按年</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdChx02">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>19</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>金额按月</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdChx03">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>19</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>金额按日</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdChx04">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>19</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>金额按收费人员</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdChx06">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>19</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>进出数量按通道</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdChx05">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>计时卡收费明细</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdChx07">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>按省统计</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdChx08">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>省内统计</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdChx09">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>月租卡信息</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdChx10">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>月租卡进出次数</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>630</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="text">
    <string>至</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>630</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="text">
    <string>查询范围：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblClose">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>7</y>
     <width>30</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dReportStartDate">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>630</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dReportEndDate">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>630</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrint">
   <property name="geometry">
    <rect>
     <x>514</x>
     <y>631</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="text">
    <string>报表打印</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGenerate">
   <property name="geometry">
    <rect>
     <x>433</x>
     <y>631</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="text">
    <string>报表生成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>595</x>
     <y>631</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
